rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -98,6 +98,7 @@ public abstract class SystemProperties {
 
     public static final String PROPERTY_PERSIST_STATES_CACHE_SNAPSHOT = "cache.states.persist-snapshot";
     public static final String PROPERTY_PERSIST_BLOOMS_CACHE_SNAPSHOT = "cache.blooms.persist-snapshot";
+    public static final String PROPERTY_PERCENTAGE_OF_PEERS_TO_CONSIDER_IN_RND_SELECTION = "rpc.percentageOfPeersToConsiderInRandomSelection";
```

Comment:
why `rpc`? how about `sync.topBest`? Also this config has to be added to reference.conf and expected.conf

Reply:
No reason to be `rpc` I don't know the whole config structure. I can move it to be under `sync`

Reply:
Addressed